src/bin/main.rs

```diff
@@ -149,20 +149,27 @@ fn benchmark() -> Result<(), Box<dyn Error>> {
 }
 
 #[derive(Parser)]
+#[command(version, about, long_about = None)]
 struct Arguments {
     /// Path to the `EE.log` file located in the game installation directory
     ///
     /// Most likely located at `~/.local/share/Steam/steamapps/compatdata/230410/pfx/drive_c/users/steamuser/AppData/Local/Warframe/EE.log`
     game_log_file_path: Option<PathBuf>,
+    /// Warframe Window Name
+    ///
+    /// some systems may require the window name to be specified (e.g. when using gamescope)
+    #[arg(short, long, default_value = "Warframe")]
+    window_name: Option<String>,
```

Comment:
Since this has a default value, it does not need to be an `Option` type to be an optional argument. Then you also don't have to unwrap it below and can just use `arguments.window_name` directly in the `find` closure.

Reply:
thanks.